### DISCUSSION TITLE
The bandolier must have slot for heavy handheld weapon (assault rifle, heavy machine gun, or arc emitter)

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14173] [Ideas and suggestions]
The bandolier cannot be worn simultaneously with an assault rifle, heavy machine gun, or arc emitter.
What's the point of having a bandoleer if you can't carry it with your weapon?
Please, lets make a slot for weapons in the bandoleer so that an assault rifle or machine gun can be placed in the bandoleer. 
Then it will make sense.
In fact, the bandoleer is a "Tactical Load-Bearing Vest".
Tell us why you can't sling an assault rifle on your back when you're wearing a "Tactical Load-Bearing Vest"?
Tactical Load-Bearing Vest + assault rifle on your back is the basis of combat equipment, is'nt it?

**Comment by Arctomachine** [comment]
If it is vest, it should go into body armor slot too, not belt.

**Comment by Smrtonosan** [comment]
It's not a load bearing vest, it's a bandolier, it's main purpose is to carry ammo, granted it looks like a "chest rig".

Bandolier has definitely strayed from the original purpose of "going into the body slot so that you are able to stow away your heavy weapon on your belt", but the positives far outweigh the negatives.

I definitely would find this useful in singleplayer, for an example, when wearing the bandolier and carrying a heavy weapon in your hands you are unable to use the plasma cutter to cut wrecked submarine's doors without dropping the weapon and leaving yourself open to creature attacks. Or for another example, on the submarine in multiplayer you cannot reload the ship mounted weapons without dropping your heavy weapon on the floor which causes the same issue.

Currently you have to decide whether or not it is worth using the bandolier with the heavy weapons depending on the situation. If you are manning the guns, it's best just to use the bandolier and a sidearm, when you are going into an abandoned outpost there is no reason not to take a heavy weapon and the bandolier as you can open the doors without dropping the weapon, especially now that you can wear armour alongside the bandolier. And so on.

It would also be strange for it to be the only toolbelt item that can also completely mitigate the negative of not being able to stow away weapons, as it isn't a load bearing vest.
